modules/phase_field/tests/grain_tracker_test/grain_tracker_ebsd.i

```diff
@@ -18,6 +18,7 @@
   [./PolycrystalICs]
     [./ReconVarIC]
       ebsd_reader = ebsd
+      advanced_op_assignment = true
```

Comment:
Oops

On Wed, Jul 27, 2016 at 5:37 PM Daniel Schwen <notifications@github.com>
wrote:

> In modules/phase_field/tests/grain_tracker_test/grain_tracker_ebsd.i
> <https://github.com/idaholab/moose/pull/7411#discussion_r72543575>:
>
> > @@ -18,6 +18,7 @@
> >    [./PolycrystalICs]
> >      [./ReconVarIC]
> >        ebsd_reader = ebsd
> > +      advanced_op_assignment = true
>
> Looks like the advanced assignment requires re-golding of the test.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/idaholab/moose/pull/7411/files/f2c0fffbaaf5829f5287000396315fb23b83975e#r72543575>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC5XICNysqmzzvRs6aP1jl0Sd7GiTSUrks5qZ-u-gaJpZM4JWs7i>
> .
>